spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -118,7 +118,8 @@
         )
         expect(phone_radio_button).to be_disabled
 
-        select 'Canada +1', from: 'International code'
+        select_country('jp')
```

Comment:
This is misleading. Did you mean to select Canada? These tests aren't really testing the behavior as a user would interact with the form. If what we want to test is both the country selection from the dropdown and then entering a number after the country code is automatically populated, then we should use `send_keys`. On the other hand, if all we want to test here is that entering a number that isn't supported shows an error message and that the message gets cleared after entering a supported phone number, then we don't need to call `select_country`. We can use `fill_in` by itself.